ElectronClient/app/app.js

```diff
@@ -384,6 +384,7 @@ class Application extends BaseApplication {
 		const importItems = [];
 		const exportItems = [];
 		const toolsItemsFirst = [];
+		var   toolsItems = [];
```

Comment:
`let` please. Spacing is not consistent with the rest of the code base but if eslint allow it, why not.

Reply:
I used the spaces to align it with the other vars. Old habit. I'll revert it, since the assignments are not aligned either.

Not sure why I used `var`. I'll fix this. 

Sorry, I'm still struggling a bit with the new build system. I haven't found my new workflow yet.